lib/@types/Identity.ts

```diff
@@ -0,0 +1,26 @@
+export enum IdentityTypes {
```

Comment:
how could we make it so we don't need to update the lib each time someone on the backend team add an identity type on connect?

This is not a question for right now but I know they won't think of the lib without some kind of process and even with that, there could be misses. So we will need to keep that in mind (in my mind, that shouldn't be something that needs to be done but I don't know yet how we could do it)

Reply:
I keep thinking about it. I think it's a quite complex task to do. We would need to generate types/enums from the GraphQL schemas (should be doable). The hard part would be to keep track of the schemas update, to re-generate the types/enums, and publish after that. I have no clear solution for that, except for a GitHub template with a checkbox regarding identities type update, and if checked, this should be reported to the team maintaining the lib. Looks like a pretty heavy and error prone solution.